src/item_contents.h

```diff
@@ -130,9 +130,6 @@ class item_contents
         /** True if every pocket is rigid or we have no pockets */
         bool all_pockets_rigid() const;
 
-        // True if every pocket is rigid. False if not or we have no pockets
```

Comment:
Seems to be different from `all_pockets_rigid` that when there are no pockets,`all_pockets_rigid` returns true while this method returns false.